tests/integration_tests/functional/test_rng.py

```diff
@@ -91,14 +91,6 @@ def test_rng_snapshot(test_microvm_with_rng, microvm_factory):
 
     new_vm = microvm_factory.build()
     new_vm.spawn()
-    iface = NetIfaceConfig()
```

Comment:
"[test: clean up network interfaces and SSH helper](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/a8fb4b7369f8ede378399a88cb42adee05b25990)" fails due to the following error:
```
integration_tests/functional/test_rng.py:94: in test_rng_snapshot
    new_vm.restore_from_snapshot(
        mem_path   = PosixPath('/srv/jailer/firecracker/ce3e34e2-e658-4a95-9b4d-abc3c5a32c1a/root/test.mem')
        microvm_factory = <conftest.microvm_factory.<locals>.MicroVMFactory object at 0x7fd72e7d7dc0>
        new_vm     = <framework.microvm.Microvm object at 0x7fd72f329780>
        snapshot_path = PosixPath('/srv/jailer/firecracker/ce3e34e2-e658-4a95-9b4d-abc3c5a32c1a/root/test.snap')
        test_microvm_with_rng = <framework.microvm.Microvm object at 0x7fd72e7d62c0>
        vm         = <framework.microvm.Microvm object at 0x7fd72e7d62c0>
framework/microvm.py:804: in restore_from_snapshot
    assert response.ok, response.content
E   AssertionError: b'{"fault_message":"Load microVM snapshot error: Failed to restore from snapshot: Failed to build microVM from snapshot: Failed to restore MMIO device: Cannot restore devices: Net(CreateNet(TapOpen(IfreqExecuteError(Os { code: 1, kind: PermissionDenied, message: \\"Operation not permitted\\" }, \\"tap0\\"))))"}'
```